src/Item/Item.service.ts

```diff
@@ -172,11 +172,7 @@ export class ItemService {
     }
   ): Promise<(ItemAttributes & { total_count: number })[]> {
     const thirdParties = await thirdPartyAPI.fetchThirdPartiesByManager(address)
-    if (thirdParties.length <= 0) {
-      return []
-    }
-
-    const thirdPartyIds = thirdParties.map((thirdParty) => thirdParty.id)
+    const thirdPartyIds = thirdParties.map((thirdParty: any) => thirdParty.id)
```

Comment:
why any here :( ?

Reply:
I forgot to delete it, it was for testing purposes.